weis/glue_code/runWEIS.py

```diff
@@ -318,7 +334,7 @@ def run_weis(fname_wt_input, fname_modeling_options, fname_opt_options, overridd
 
             if blade_constraints['tip_deflection']['flag']:
                 if blade_opt_options['structure']['spar_cap_ss']['flag'] or blade_opt_options['structure']['spar_cap_ps']['flag']:
-                    wt_opt.model.add_constraint('tcons.tip_deflection_ratio', upper= 1.0)
+                    wt_opt.model.add_constraint('tcons.tip_deflection_ratio', upper= 0.8)
```

Comment:
Just double-checking: is this supposed to be 0.8 here?

Reply:
@ptrbortolotti - thoughts?

Reply:
@nikhar-abbas I believe this should be a user input coming in from the analysis option yaml. I did fix in wisdom and I forgot to do the same in WEIS: https://github.com/WISDEM/WISDEM/blob/d9ec4ea295ecebb597998894d7ccaeaeb7de406f/wisdem/glue_code/runWISDEM.py#L245
Would you mind fixing it in your PR? thanks

Reply:
Okay, I changed this in runWeis.py and in all of the analysis_options.yaml files in the examples folder: https://github.com/WISDEM/WEIS/pull/16/commits/3b8d61461d2f4a27fa077a4b32c678a12ee9366a